packages/web-frontend/cypress/config/dashboardReportUrls/lesmis.json

```diff
@@ -1,25 +1,25 @@
 [
   {
-    "code": "LESMIS_repetition_rate_lowerSecondary_district",
+    "code": "LESMIS_age_of_grade1_entrance",
     "project": "laos_schools",
     "orgUnit": "LA_Phouvong District",
-    "dashboard": "Student Outcomes",
+    "dashboard": "Student Enrolment",
     "startDate": "2021-01-01",
     "endDate": "2021-12-31"
   },
   {
-    "code": "LESMIS_repetition_rate_upperSecondary_district",
+    "code": "LESMIS_age_of_grade1_entrance",
     "project": "laos_schools",
-    "orgUnit": "LA_Phouvong District",
-    "dashboard": "Student Outcomes",
+    "orgUnit": "LA_Attapue",
+    "dashboard": "Student%20Enrolment",

```

Comment:
In the above two json configs one of the `"dashboard": "Student%20Enrolment",` one of the dashboard's have `20%` while the other does not any particular reason?

Reply:
Oops good catch Bindu - `20%` is an encoded html space. Both should work but using a space is more human-friendly, so I'm gonna change that